src/styles/_tabs.scss

```diff
@@ -0,0 +1,57 @@
+@import 'variables';
+
+.react-tabs {
+  -webkit-tap-highlight-color: transparent;
+}
+
+.react-tabs__tab-list {
+  margin: 0 0 10px;
+  padding: 30px 0 0 30px;
+  background-color: $gray-lighter;
+}
+
+.react-tabs__tab {
+  display: inline-block;
+  border-bottom: none;
+  bottom: -1px;
+  position: relative;
+  list-style: none;
+  padding: 6px 12px;
+  cursor: pointer;
+}
+
+.react-tabs__tab--selected {
+  background: #fff;
+  border-color: #aaa;
+  color: black;
+  border-top: 5px solid $blue;
+}
+
+.react-tabs__tab--disabled {
+  color: GrayText;
+  cursor: default;
+}
+
+.react-tabs__tab:focus {
+  box-shadow: 0 0 5px hsl(208, 99%, 50%);
+  border-color: hsl(208, 99%, 50%);
```

Comment:
Same with this color.  it is repeated here so it probably belongs with the other color variables.

Reply:
Agreed. This is part of Dylan's work and I'm not actually sure what it is styling. Could you tell me what color it is and I will make it a variable?